bin/web/svelte.config.js

```diff
@@ -1,20 +1,23 @@
 import adapter from '@sveltejs/adapter-static';
 
 export default {
-	kit: {
-		adapter: adapter({
-			// default options are shown. On some platforms
-			// these options are set automatically — see below
-			pages: 'build',
-			assets: 'build',
-			fallback: undefined,
-			precompress: false,
-			strict: true
-		}),
-		version: {
+    kit: {
+        adapter: adapter({
+            // default options are shown. On some platforms
+            // these options are set automatically — see below
+            pages: 'build',
+            assets: 'build',
+            fallback: undefined,
+            precompress: false,
+            strict: true
+        }),
+        output: {
+            bundleStrategy: 'inline'
```

Comment:
is this what lets us package the single `index.html` file? might be worth adding a comment to that effect

Reply:
done